phoenix-core/src/main/java/org/apache/phoenix/util/ScanUtil.java

```diff
@@ -957,4 +971,222 @@ public static int getClientVersion(Scan scan) {
     public static void setClientVersion(Scan scan, int version) {
         scan.setAttribute(BaseScannerRegionObserver.CLIENT_VERSION, Bytes.toBytes(version));
     }
+
+    public static boolean isServerSideMaskingEnabled(PhoenixConnection phoenixConnection) {
+        String isServerSideMaskingSet = phoenixConnection.getClientInfo(
+                QueryServices.PHOENIX_TTL_SERVER_SIDE_MASKING_ENABLED);
+        boolean isServerSideMaskingEnabled = phoenixConnection.getQueryServices()
+                .getConfiguration().getBoolean(
+                        QueryServices.PHOENIX_TTL_SERVER_SIDE_MASKING_ENABLED,
+                        QueryServicesOptions.DEFAULT_SERVER_SIDE_MASKING_ENABLED) ||
+                ((isServerSideMaskingSet != null) && (Boolean.parseBoolean(isServerSideMaskingSet)));
+        return isServerSideMaskingEnabled;
+
+    }
+
+    public static long getPhoenixTTL(Scan scan) {
```

Comment:
Can we add some unit tests for these newly added methods? Some diffs are from copy-pasting from IndexUtil, so no new tests for those is fine, but would be nice to have tests for new ones.